ykman-gui/py/yubikey.py

```diff
@@ -593,7 +593,11 @@ def piv_import_file(self, slot, file_url, password=None,
 
                     controller.import_certificate(
                             SLOT[slot], cert_to_import)
-        return success()
+        return {
+            'success': True,
+            'imported_cert': is_cert,
+            'imported_key': is_private_key
+        }
```

Comment:
This could also be written as
```
        return success({
            'imported_cert': is_cert,
            'imported_key': is_private_key
        })